docs/Configuration/Deployment Troubleshooting.md

```diff
@@ -0,0 +1,113 @@
+Deployment Troubleshooting
+
+## Diagnostics and Logging
+
+Navigating to:
+
+*  [http://localhost:8080/snapshot](http://localhost:8080/snapshot) will produce a zip file suitable for support diagnosis
+
+*  [http://localhost:8080/web/admin/log.html](http://localhost:8080/web/admin/log.html) to review the current logs. This will also update with new entries.
+
+*  [http://localhost:8080/snapshot/threads](http://localhost:8080/snapshot/threads) to review thread info and CPU usage.
+
+*  [http://localhost:8080/health](http://localhost:8080/health) to check general health.
+
+## Log Levels
```

Comment:
@Egis-Moshe - Could not find this in the existing logging.md.  A couple of links were in CPU Usage.md thought. Should I remove this part entirely then?